src/subdomains/generic/user/models/bank-data/bank-data.entity.ts

```diff
@@ -122,4 +123,23 @@ export class BankData extends IEntity {
 
     return [this.id, update];
   }
+
+  internalReview(comment?: string): UpdateResult<BankData> {
+    const update: Partial<BankData> = {
+      status: ReviewStatus.INTERNAL_REVIEW,
+      comment,
+    };
+
+    Object.assign(this, update);
+
+    return [this.id, update];
+  }
+
+  get isInReview(): boolean {
+    return [ReviewStatus.MANUAL_REVIEW, ReviewStatus.INTERNAL_REVIEW].includes(this.status);
+  }
+
+  get isCompleted(): boolean {
+    return this.status === ReviewStatus.COMPLETED;
+  }
```

Comment:
Maybe `isApproved` or `isReviewed` or similar would be better? A bank data cannot be completed IMO.